### ISSUE TITLE
Publish to package repositories

### ISSUE THREAD

**Issue by sevenbitbyte** [mcgill-robotics/ros-teensy#2]
This is a wonderful package, it should be published to the package repositories to enable easier consumption.

**Comment by anassinator** [comment]
Agreed. We have a few packages that would benefit from this, so I'll look into it when I get a chance.

**Comment by doisyg** [comment]
Same here, would love to see this package released

**Comment by sevenbitbyte** [comment]
Oh man, totally forgot about this. Something tells me most of the maintainers have probably moved on to other things. I'd be happy to fork and release from my work org ( https://github/roshub ). Also I'll pose a question in ros-drivers (https://github.com/ros-drivers/rosserial) to see if they'd like to release as I think this fits there.

**Comment by sevenbitbyte** [comment]
@dlavoieb and @anassinator what are your thoughts on releasing from this org vs ros-serial vs roshub?

We should get this released so people can use it easily.

**Comment by dlavoieb** [comment]
I would be more than happy to see this get released. As @sevenbitbyte pointed out however, I have moved on and not working on the project anymore.
If anyone is willing to put in the work needed to release this package, you have my blessings, have fun!

**Comment by anassinator** [comment]
@yasasa: Thoughts on releasing this package along with a few others in this org? Ideally this could be integrated into our CI. If that's too much work, someone else could release this package for now.

**Comment by sevenbitbyte** [comment]
Thanks @dlavoieb I've posted an initial PR -> https://github.com/ros-drivers/rosserial/pull/417

**Comment by anassinator** [comment]
@sevenbitbyte thanks!
